### ISSUE TITLE
 vals = torch.mean(vals, keepdim=True)

### ISSUE THREAD

**Issue by 0xTechSavvy** [nnUyi/PGGAN#2]
need dim?

**Comment by nnUyi** [comment]
you can delete it, it doesn't matter @hongdayu
